package.nls.json

```diff
@@ -112,6 +112,7 @@
             "Markdown text between `` should not be translated or localized (they represent literal text) and the capitalization, spacing, and punctuation (including the ``) should not be altered."
         ]
     }, 
+	"cmake-tools.configuration.cmake.ctest.testSuiteDelimiter.debugLaunchTarget": "Target name from launch.json to start when debugging a test with CTest. By default and in case of a non-existing target, this will show a picker with all available targets.",
```

Comment:
```suggestion
	"cmake-tools.configuration.cmake.ctest.debugLaunchTarget": "Target name from launch.json to start when debugging a test with CTest. By default and in case of a non-existing target, this will show a picker with all available targets.",
```